app/templates/view_users.html

```diff
@@ -128,8 +108,35 @@ <h1 class="govuk-heading-xl">Find a user</h1>
             Deactivated
           {% endif %}
         {% endif %}
-      {% endcall %}
-
-    {% endcall %}
-  {% endcall %}
+      {% endset %}
+
+      {% set row = ns.rows.append(
+        [
+          {"text": user.name},
+          {"text": user.role},
+          {"html": supplier},
+          {"html": logged_in_at},
+          {"html": password_changed_at},
+          {"html": locked},
+          {"html": change_status},
+        ]
+      ) %}
+    {% endfor %}
+
+    {{ govukTable({
+      'head': [
+        {'text': 'Name'},
+        {'text': 'Role'},
+        {'text': 'Supplier'},
+        {'text': 'Last login'},
+        {'text': 'Last password change'},
+        {'text': 'Locked'},
+        {'text': 'Change status'},
+      ],
+      'rows': ns.rows,
+      "classes": "dm-table govuk-!-margin-bottom-6",
```

Comment:
I don't think `dm-table` is strictly necessary here if we aren't doing any custom styling - the only use of it I can find is for an override on supplier frontend: https://github.com/alphagov/digitalmarketplace-supplier-frontend/blob/8022f84b54d89b8a877cac894ffa68289c3d1a97/app/assets/scss/overrides/_table.scss

Reply:
Do we even need the `govuk-!-margin-bottom-6`? It doesn't seem to make any difference if I remove it...

Reply:
Good catch - `govukTable` includes that margin by default: https://github.com/alphagov/govuk-frontend/blob/6cf094045edb4293fcc9531a955787f08ff8f8a4/src/govuk/components/table/_index.scss#L6